rgw/v2/tests/s3cmd/test_header_size.py

```diff
@@ -0,0 +1,148 @@
+"""
+test_header_size - Test to verify header size in Beast front end using s3cmd
+
+Usage: test_header_size.py -c <input_yaml>
+
+<input_yaml>
+    Note: Following yaml can be used
+    test_header_size.yaml
+
+Operation:
+    Create an user
+    Create a bucket with user credentials
+    Performs object upload with header
+    Verify object for its header
+"""
+
+
+import argparse
+import json
+import logging
+import os
+import random
+import sys
+import traceback
+from string import ascii_lowercase, ascii_uppercase, digits
+
+sys.path.append(os.path.abspath(os.path.join(__file__, "../../../..")))
+
+
+from v2.lib import resource_op
+from v2.lib.exceptions import RGWBaseException, TestExecError
+from v2.lib.s3.write_io_info import BasicIOInfoStructure, IOInfoInitialize
+from v2.lib.s3cmd import auth as s3_auth
+from v2.tests.s3cmd import reusable as s3cmd_reusable
+from v2.utils import utils
+from v2.utils.log import configure_logging
+from v2.utils.test_desc import AddTestInfo
+
+log = logging.getLogger()
+TEST_DATA_PATH = None
+
+
+def test_exec(config):
+    """
+    Executes test based on configuration passed
+    Args:
+        config(object): Test configuration
+    """
+    io_info_initialize = IOInfoInitialize()
+    basic_io_structure = BasicIOInfoStructure()
+    io_info_initialize.initialize(basic_io_structure.initial())
+    user_info = resource_op.create_users(no_of_users_to_create=config.user_count)
+    for user in user_info:
+        user_name = user["user_id"]
+
+        ip_and_port = s3cmd_reusable.get_rgw_ip_and_port()
+        s3_auth.do_auth(user, ip_and_port)
+
+        if config.header_size:
```

Comment:
we could either return if `header_size` is not provided or use a method holding the below lines. It helps with the visuals... also we can improve readability...

Reply:
Updated as per comment